src/main/features/response/routes/check-and-send.ts

```diff
@@ -100,6 +100,7 @@ export default express.Router()
     ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
       const claim: Claim = res.locals.claim
       const draft: Draft<ResponseDraft> = res.locals.responseDraft
+      const mediationDraft: Draft<MediationDraft> = res.locals.mediationDraft
```

Comment:
I see you've added the handler to deserialise a mediationDraft in the `index.ts` file, but I can't see where we are putting it into res.locals. Am I blind or are we missing that? :O

Reply:
It should be done it the mediation index.ts but I've just checked and it is just setting .draft and not .mediationDraft... but it still works.. must be being set in the draftstore middleware perhaps?

Reply:
@simad as discussed sometime ago, its not related to this PR but i would like to know answer to your question too ... please let me know when you get a chance to explore and find answer, I will do the same...